core/src/main/java/io/questdb/cairo/CairoEngine.java

```diff
@@ -70,19 +70,21 @@ public class CairoEngine implements Closeable, WriterSource, WalWriterSource {
     private final AtomicLong asyncCommandCorrelationId = new AtomicLong();
     private final IDGenerator tableIdGenerator;
     private final TableRegistry tableRegistry;
-
     private final TextImportExecutionContext textImportExecutionContext;
+    private final ThreadSafeObjectPool<SqlCompiler> sqlCompilerPool;
+    private final AtomicLong failedWalTxnCount = new AtomicLong(1);
```

Comment:
added 

```
    // initial value of unpublishedWalTxnCount is 1 because we want to scan for unapplied WAL transactions on startup
    private final AtomicLong unpublishedWalTxnCount = new AtomicLong(1);
```